agent/xds/testdata/listeners/terminating-gateway-custom-and-tagged-addresses.envoy-1-18-x.golden

```diff
@@ -32,7 +32,7 @@
               "name": "envoy.filters.network.tcp_proxy",
               "typedConfig": {
                 "@type": "type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy",
-                "statPrefix": "terminating_gateway.default.api.foo",
+                "statPrefix": "upstream.api.default.dc1.",
```

Comment:
is this supposed to have a trailing dot?

Reply:
Doesn't look like it. Just tested this without it and got the same metrics names.